packages/cli/src/lib/cli/cliCommand.ts

```diff
@@ -4,15 +4,17 @@ import type { Client as StatesClient } from '@iobroker/db-states-redis';
 export type DBConnectCallback = (
     objects: ObjectsClient,
     states: StatesClient,
-    isOffline?: boolean,
-    objectsDBType?: string
+    isOffline: boolean,
+    objectsDBType: string,
+    /** the iobroker.json config file */
+    config?: Record<string, any>
 ) => void;
```

Comment:
I think it would be nice to have a single callback return argument with an object containing those values. That way you can just pick the value you need instead of having to look up their order and select the correct ones.